priv/react_app/src/store/reducers/CompaniesReducer.js

```diff
@@ -18,19 +18,18 @@ const companies = (state = initialState, action) => {
         const company = companies[key];
         return {"name": company.name, "id": company.id, "email": company.email}
       })
-      const newState = companyData;
+      const newState = {companies: companyData, fetching: false };
       return newState;
 
     case actionTypes.FETCH_COMPANIES_FAILURE:
-      return state;
+      return {companies: [], fetching: false };
```

Comment:
Detta gör att alla cachade companies raderas om en förfrågan misslyckas

Reply:
Ett fail ändrar inte datan vi har. Den ändrar bara att vi fick ett svar så jag ser på det

Reply:
Om `a = data_från_normalize` så borde det bara vara att merga in `a.entities` med state.